webui/module/Restore/src/Restore/Model/RestoreModel.php

```diff
@@ -155,7 +155,7 @@ public function getFiles(&$bsock=null, $jobid=null, $pathid=null) {
     */
    public function getFileVersions(&$bsock=null, $clientname=null, $pathid=null, $filename=null) {
       if(isset($bsock)) {
-         $cmd = '.bvfs_versions jobid=0 client='.$clientname.' pathid='.$pathid.' fname='.$filename;
+         $cmd = '.bvfs_versions jobid=0 client='.$clientname.' pathid='.$pathid.' fname="'.$filename.'"';
```

Comment:
```suggestion
         $cmd = '.bvfs_versions jobid=0 client='.$clientname.' pathid='.$pathid.' fname="'.addslashes($filename).'"';
```
We should add backslashes before characters that need to be escaped in the fname string, too.